site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -353,10 +351,10 @@ The content script will be injected into a page if both of the following are tru
 Because the `matches` property is required, `exclude_matches`, `include_globs`, and `exclude_globs`
 can only be used to limit which pages will be affected.
 
-The following extension injects the content script into **https://www.nytimes.com/ health**
-but not into **https://www.nytimes.com/ business** .
+The following extension injects the content script into **https://www.nytimes.com/health**
+but not into **https://www.nytimes.com/business** .
```

Comment:
The spaces were likely deliberately so that these do not create unwanted traffic for the New York Times and so that we don't have to make sure they always work. You should probably change them back, but I'm open to suggestion if you have another idea.